src/proteomes/components/styles/busco-colors.scss

```diff
@@ -0,0 +1,4 @@
+$s-color: #bce875; // completeSingle
+$d-color: #0ba5a6; // completeDuplicated
+$f-color: #eecba4; // fragmented
+$m-color: #314bb4; // missing
```

Comment:
To reduce dependency on SASS it might be good to use CSS Custom Properties here.
You might put them into the `:root` selector and make sure to import this file from both CSS files where these are used.
Or you could merge both CSS files and define them there. These could be merged because although labels and graph might be used independently in the page, they're appearing on the sames pages so it won't be loading extra CSS unnecessarily.